src/ContaoCoreBundle.php

```diff
@@ -59,5 +61,10 @@ public function build(ContainerBuilder $container)
 
         $container->addCompilerPass(new AddResourcesPathsPass());
         $container->addCompilerPass(new SetApplicationPass());
+
+        $container->addCompilerPass(
+            new CreateInitializerEventDispatcherPass(),
+            PassConfig::TYPE_BEFORE_REMOVING
```

Comment:
Any reason why you use `TYPE_BEFORE_REMOVING` and if so, can you add a one-liner as a comment in the code? :)